ci/deploy.yml

```diff
@@ -80,3 +80,37 @@ msg-system deploy dev:
     - deploy
   script:
     - aws s3 cp packages/suite-data/files/message-system/config.v1.jws s3://data.trezor.io/config/develop/config.v1.jws
+
+# npm packages deploy to npm registry
+.npm_deploy_rules: &npm_deploy_rules
+  when: manual
+  tags:
+    - deploy
+  before_script:
+    - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+  after_script:
+    - rm ~/.npmrc
+
+blockchain-link deploy npm:
+  stage: deploy to dev
+  only:
+    <<: *run_everything_rules
```

Comment:
I have just realized there is no reason not to include these three lines into the anchor as well. Sorry, last thing! Feel free to force-push then.

Reply:
Didn't put the anchor there as we want to use it for beta jobs as well and it would prevent them from running on any branch.